playbooks/roles/mattermail/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+- name: create mattermail group
+  group:
+    name: mattermail
+
+- name: create mattermail user
+  user:
+    name: mattermail
+    group: mattermail
+    home: "{{ MATTERMAIL_HOME }}"
+
+- name: Install git package
+  apt:
+    name: git
+    update_cache: yes
+
+- name: Install golang package
+  apt:
+    name: golang
```

Comment:
You can merge this with the above as a list, e.g.:

```yml
- name: Install apt packages
  apt:
    name:
      - git
      - golang
    update_cache: yes
```

Although, I think we always install git through `COMMON_SERVER_DEPENDENCIES` and thus the `common-server` role, but up to you if you want to keep it.

Reply:
Yeah, I kept it for a test playbook on Vagrant. I'll test the chat playbook and remove this.